content/projects/gsoc/index.adoc

```diff
@@ -84,53 +89,35 @@ project proposal questions and discussions, process and timeline related questio
 difficulties with mentors, students, or org admins, please use the admin mailing list shown link:#orgadmin[below].
 
 1. To start, you need a Google or Gmail account
-2. After clicking above link, a "Sign in" button appears in top-right of page
-3. After sign in, a "Join group" button appears in middle of page
+2. After clicking above the link, a "Sign in" button appears in top-right of the page
```

Comment:
I think you intend to have the user click the link that was mentioned previously in the text.  It may be easier if we assume they interpet "the link" to mean the link that was mentioned in the prior sentences.
```suggestion
2. After clicking the link, a "Sign in" button appears in the top-right of the page
```